validator/otel.go

```diff
@@ -0,0 +1,7 @@
+package validator
+
+import "go.opentelemetry.io/otel"
+
+const packageName = "github.com/evergreen-ci/evergreen/validator"
```

Comment:
should we maybe move `github.com/evergreen-ci/evergreen/` to a global const somewhere?